packages/cf-style-container/src/index.js

```diff
@@ -0,0 +1,12 @@
+import {
+	createComponent as createFelaComponent
+} from 'react-fela';
+
+export const createComponent = (rule, type = 'div', passThroughProps = []) =>
```

Comment:
That's what fela is using now so I want to match their API/terminology if possible.  Node / Element has meaning in DOM terminology. This actually accept React components as well.